test/test.js

```diff
@@ -761,6 +761,52 @@ test( "elementValue() finds radios/checkboxes only within the current form", fun
 	ok( !v.elementValue( foreignRadio ) );
 } );
 
+test( "elementValue() returns the file input's name without the prefix 'C:\\fakepath\\' ", function() {
+	var v = $( "#userForm" ).validate(),
+		fileInput = {
+			name: "fakeFile",
+			type: "file",
+			files: {},
+			nodeName: "INPUT",
+			value: "C:\\fakepath\\somefile.txt",
+			form: $( "#userForm" )[ 0 ],
+			hasAttribute: function() { return false; }
+		},
+		oldRules = $.fn.rules,
+		oldvalidationTargetFor = v.validationTargetFor;
+
+	equal( v.elementValue( fileInput ), "somefile.txt" );
+
+	$.fn.rules = function() {
+		return {
+			minlength: 10
+		};
+	};
+
+	v.validationTargetFor = function() {
+		return fileInput;
+	};
+
+	ok( v.check( fileInput ), "The fake file input is valid (length = 12, minlength = 10)" );
```

Comment:
Is there a way to validate an element without showing the error ?
I guess the only the solution I have is to use undocumented api.